App/src/Repository/ExpenseRepository.php

```diff
@@ -6,19 +6,31 @@
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Doctrine\Persistence\ManagerRegistry;
 
+/**
+ * @extends ServiceEntityRepository<Expense>
+ */
 class ExpenseRepository extends ServiceEntityRepository
 {
     public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, Expense::class);
     }
 
+    /**
+     * @param int $groupId
+     * @return Expense[]
```

Comment:
PHP CS Fixer ajoute un saut de ligne avant l'annotation `@return` 👀 
```suggestion
     * @param int $groupId

     * @return Expense[]
```

Reply:
done